### PR TITLE
Add lint warning for using default pings with SDKs that don't provide them

### PR DESCRIPTION
**Pull request by brizental**
### Pull Request checklist ###

- [x] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry to `CHANGELOG.md` or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

File: glean_parser/lint.py [hunk 1]
```diff
@@ -225,6 +225,28 @@ def check_valid_in_baseline(
         )
 
 
+def check_default_in_pings(
+    metric: metrics.Metric, parser_config: Dict[str, Any]
+) -> LintGenerator:
+    output_format = parser_config.get("output_format")
```
[the remark this answers is not in the document]
Comment (brizental) [reply glean_parser/lint.py:231]: The glinter command doesn't get an output format, so there is no way to run this lint on it. I am fine just dropping this lint as whole if you see it as a big deal. Adding another option to the glinter to know the output format is overkill just for this lint.

[the remark this answers is not in the document]
Comment (brizental) [reply glean_parser/lint.py:231]: Re-requist review specifically to get an answer here ^^. No strong opinions on it, however if we are to close this PR then I won't attend the other comments :)

[the remark this answers is not in the document]
Comment (badboy) [reply glean_parser/lint.py:231]: Oh, sorry, I somehow missed your earlier reply here.

I'm leaning toward dropping this lint.
Having different lints between `translate` and `glinter` still seems weird to me (I assume we're not already having one like these?)

Did we already have any stumble upon this? That would be a strong signal that we _should_ have the lint.

[the remark this answers is not in the document]
Comment (brizental) [reply glean_parser/lint.py:231]: Nope, never ran into it. Closing the PR.

**Pull request closed without merging**